test/spec/ol/format/wfs.test.js

```diff
@@ -686,13 +687,55 @@ describe('ol.format.WFS', function () {
       expect(serialized.firstElementChild).to.xmleql(parse(text));
     });
 
+    it('creates a dwithin filter', function () {
+      const text =
+        '<wfs:Query xmlns:wfs="http://www.opengis.net/wfs" ' +
+        '    typeName="area" srsName="EPSG:4326" ' +
+        '    xmlns:topp="http://www.openplans.org/topp">' +
+        '  <ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">' +
+        '    <ogc:DWithin>' +
+        '      <ogc:PropertyName>the_geom</ogc:PropertyName>' +
+        '      <gml:Polygon xmlns:gml="http://www.opengis.net/gml">' +
+        '        <gml:exterior>' +
+        '          <gml:LinearRing>' +
+        '            <gml:posList srsDimension="2">' +
+        '              10 20 10 25 15 25 15 20 10 20' +
+        '            </gml:posList>' +
+        '          </gml:LinearRing>' +
+        '        </gml:exterior>' +
+        '      </gml:Polygon>' +
+        '      <ogc:Distance units="m">10</ogc:Distance>' +
+        '    </ogc:DWithin>' +
+        '  </ogc:Filter>' +
+        '</wfs:Query>';
+      const serialized = new WFS().writeGetFeature({
+        srsName: 'EPSG:4326',
+        featureTypes: ['area'],
+        filter: dwithinFilter(
+          'the_geom',
+          new Polygon([
+            [
+              [10, 20],
+              [10, 25],
+              [15, 25],
+              [15, 20],
+              [10, 20],
+            ],
+          ]),
+          10,
+          'm'
+        ),
+      });
+      expect(serialized.firstElementChild).to.xmleql(parse(text));
+    });
+
     it('creates During property filter', function () {
       const text =
         '<wfs:Query xmlns:wfs="http://www.opengis.net/wfs" ' +
         '    typeName="states" srsName="EPSG:4326">' +
         '  <ogc:Filter xmlns:ogc="http://www.opengis.net/ogc">' +
         '    <ogc:During>' +
-        '      <fes:ValueReference xmlns:fes="http://www.opengis.net/fes">date_prop</fes:ValueReference>' +
+        '      <ogc:PropertyName>date_prop</ogc:PropertyName>' +
```

Comment:
Ideally, you'd modify this test case to use WFS 2.0 (and continue using ValueReference), and add another using WFS 1.x (with PropertyName).

Reply:
Thanks @ahocevar  for reviewing, 

The during test for wfs 2.0 is already there.

The during implementation was a bit weird the filter is only specified in the 2.0.0 filter encoding. But the filter was already there for the wfs 1.x format but with a mix of the ogc and fes namespace. 

To handle this there are three options
- remove during from wfs 1.0
- change wfs 1.0 during implementation to use ogc:propertyName for consistency ( This is already done)
- keep wfs 1.0 during implementation as it was with using fes:valueReference instead of ogc:propertyName for backwards compatibility

What implementation do you prefer also see: https://github.com/openlayers/openlayers/pull/6786

Reply:
Thanks for the explanations @jbo023. I think users might be relying on the existing implementation, so if it doesn't mean adding weird code, I'd opt for keeping the WFS 1.x `During` implementation as it was for backwards compatibility.

Reply:
@ahocevar ok, i changed the behaviour so that the during filter with wfs 1.0 keeps the old behaviour